follower-read: refine some terms (#2281)
pingcap/docs: reference/performance/follower-read.md

### Follower replica selection strategy

> Because the Follower Read feature will not damage the transaction isolation level of Snapshot Isolation of TiDB, TiDB adopts the round-robin strategy to select the follower replica. Currently, for the coprocessor requests, the granularity of the Follower Read load balancing policy is at the connection level. For a TiDB client connected to a specific Region, the selected follower is fixed, and is switched only when it fails or the scheduling policy is adjusted.

Reviewer: ```suggestion
Because the Follower Read feature does not break TiDB's Snapshot Isolation transaction isolation level, TiDB adopts the round-robin strategy to select the follower replica. Currently, for the coprocessor requests, the granularity of the Follower Read load balancing policy is at the connection level. For a TiDB client connected to a specific Region, the selected follower is fixed, and is switched only when it fails or the scheduling policy is adjusted.
```
